packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -578,15 +589,25 @@ export class Combobox extends FormAssociatedCombobox {
     }
 
     /**
-     * Focus and select the content of the control based on the first selected option.
+     * Focus and set the content of the control based on the first selected option.
      *
-     * @param start - The index for the starting range
      * @internal
      */
-    private setInlineSelection(): void {
+    private setInputToSelection(): void {
```

Comment:
I made this new private method so that we maintain current behavior for things like pressing 'Tab', where it would just put the cursor at the end of the selected item from the popup (that was navigated to via arrow keys), and not actually select the text.